samples/drivers/rtc/ds1307/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
A dedicated LICENSE file is not needed, files can just contain a SPDX license header at the top, you can find this in all other intree files. Also locally you can use checkpatch to test your changes before submitting.